scripts/dockerfiles/Dockerfile.alpine

```diff
@@ -1,6 +1,7 @@
 ARG GO_VERSION
 FROM golang:$GO_VERSION-alpine3.10 as builder
 # hadolint ignore=DL3018
+RUN addgroup -S 10001 && adduser -S agentuser -G 10001
```

Comment:
I think This line should be at line 13, this is not running container.

Reply:
you need to add the user here and then get it from the builder